src/registrar/assets/src/sass/_theme/_header.scss

```
@@ -413,3 +413,16 @@ $-expand-less-icon: (
         }
     }
 }
+
+// Extended header only displaying topmost navigation
+.usa-header__top-level-only {
+    border-bottom-width: 0;
+
+    .usa-nav {
+        background-color: white !important;
```

Overrides USWDS defaults to hide the primary nav on the Your portfolios page